client/src/locales/en.json

```diff
@@ -167,6 +167,7 @@
   "list.items-container.filter-box": "Filter by:",
   "list.items-container.sort-box": "Sort by:",
   "list.items-container.unhandled": "Unhandled",
+  "list.items-counter-label": "Showing {displayedItemsCount} of {totalItemsCount} items",
```

Comment:
Please investigate and use plural rules (test  case 1/1).

Reply:
use https://github.com/formatjs/react-intl/blob/master/docs/Components.md#message-syntax